startup-scripts/sysconfig-downtimed

```diff
@@ -0,0 +1,45 @@
+# Install in/as /etc/sysconfig/downtimed
+#
+OPTIONS=""
+# -D
+#  Do not create nor update the downtime database.
+#
+# -d datadir
+#  The directory where the time stamp files as well as the downtime database
+#  are located. The default directory is determined at compile time.
+#
+# -F
+#  Do not call daemon(3) to fork(2) to background. Useful with modern
+#  system service managers such as systemd(8), launchd(8) and others.
+OPTIONS="$OPTIONS -F "
```

Comment:
This adds `-F` option as a setting which is enabled by default. However this is not mentioned in the commit message. Is it intentional?

I thought that on RHEL 6 the init script function `daemon` expects the process to `fork()` on its own. Is this not the case?

Reply:
Actually, that was an oversight of mine.

I had meant to comment out 14 leaving it to show a means for concatenating options.